.github/workflows/pr_scan.yml

```diff
@@ -70,6 +71,7 @@ jobs:
 
     - name: SwiftLint
       run: |
+        brew install swiftlint
```

Comment:
"macos-13" runner [supposed to have](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#linters) swiftlint

Reply:
It does not tho

Reply:
Apparently the arm version of the runner doesn't have the linter installed 🤷  https://github.com/actions/runner-images/blob/main/images/macos/macos-13-arm64-Readme.md